models/meirelles2024credit-operations/ontology.ttl

```diff
@@ -0,0 +1,1110 @@
+@prefix : <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/>.
+@prefix ontouml: <https://w3id.org/ontouml#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix owl: <http://www.w3.org/2002/07/owl#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+
+<https://w3id.org/ontouml-models/model/meirelles2024credit-operations/> rdf:type ontouml:Project;
+    ontouml:name "ontologia"@en;
+    ontouml:model :zcju0_GGAqAkfgBO_root;
+    ontouml:diagram :nwoe0_GGAqAkfgWd.
+:zcju0_GGAqAkfgBO_root rdf:type ontouml:Package;
+    ontouml:project :zcju0_GGAqAkfgBO;
+    ontouml:name "ontologia"@en;
+    ontouml:containsModelElement :FXfh0_GGAqAkfgZI, :jHYx0_GGAqAkfgaw, :Vf8x0_GGAqAkfgbP, :wgzR0_GGAqAkfgZ2, <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/.FZl0_GGAqAkfgma>, :csQt0_GGAqAkfgpb, :dbKd0_GGAqAkfgrO, :zu8D0_GGAqAkfgtE, :edhT0_GGAqAkfgwf, <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/50Jz0_GGAqAkfgxy>, :AGVr0_GGAqAkfg4V, :r9N70_GGAqAkfg6Q, :yJ3n0_GGAqAkfg7v, <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/c_a30_GGAqAkfg.l>, :SWWv0_GGAqAkfhEc, <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/E_5yM_GGAqAkfhO.>, :PUlyM_GGAqAkfhPN, :PCdyM_GGAqAkfhPb, :jIDx0_GGAqAkfgby, :Ekn10_GGAqAkfgn3, <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/B.1t0_GGAqAkfgqP>, :xp5d0_GGAqAkfgsL, :UbDj0_GGAqAkfgu8, :GRCz0_GGAqAkfgxU, <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/7Xab0_GGAqAkfg5a>, <https://w3id.org/ontouml-models/model/meirelles2024credit-operations/6fIn0_GGAqAkfg7A>, :Ss1X0_GGAqAkfg9u, :IRtP0_GGAqAkfhCA, :RwbgM_GGAqAkfhIE, :RXaqM_GGAqAkfhUy.
+:FXfh0_GGAqAkfgZI rdf:type ontouml:Class;
+    ontouml:project :zcju0_GGAqAkfgBO;
+    ontouml:name "Opera��o de Cr�dito Agregada"@en;
```

Comment:
There something wrong with the name of some of theses classes. I suppose it is an encoding issue?

Reply:
However, you just made a single comment. Was that all?